libraries/node/consensus/src/transaction_manager/transaction_manager.cpp

```diff
@@ -121,16 +116,6 @@ std::pair<bool, std::string> TransactionManager::insertTransaction(const Transac
     }
   }
 
-  // TODO: new pending tx could be omitted here ? Currently we ommit new pending tx only if it is included in dag block
-  // TODO: either create different events, e.g. new pending vs new accepted or remove this whole TODO
-  //  transaction_accepted_.emit(trx.getHash());
-
-  if (broadcast == true) {
-    if (auto net = network_.lock(); net && conf_.network.network_transaction_interval == 0) {
-      net->onNewTransactions({trx});
```

Comment:
where is this functionality now?

Reply:
According to configs there is no situations when `network_transaction_interval` should be zero. So it is kind of unreachable code

Reply:
ok, I see